internal/uvm/network.go

```diff
@@ -55,6 +76,25 @@ func (uvm *UtilityVM) RemoveNetNS(id string) error {
 		err = uvm.removeNamespaceNICs(ns)
 		delete(uvm.namespaces, id)
 	}
+
+	// Add a Guest Network namespace
+	if uvm.operatingSystem == "windows" {
+		hcnNamespace, err := hcn.GetNamespaceByID(id)
+		if err != nil {
+			return err
```

Comment:
Should we fail here or just continue, if error is not found

Reply:
We should fail. But this logic needs to happen post decrement of the ref count. If we don't actually remove it that would be bad.

Reply:
Pending.